testing/log.go

```diff
@@ -41,7 +41,7 @@ type containLogMessageMatcher struct {
 }
 
 func (matcher *containLogMessageMatcher) Match(actual interface{}) (success bool, err error) {
-	entries := actual.([]*logrus.Entry)
+	entries := actual.([]logrus.Entry)
```

Comment:
Is this change required?

Reply:
Yes, it breaks the tests because one of the updates was the logrus